cf-custom-resources/lib/custom-domain-app-runner.js

```diff
@@ -0,0 +1,235 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/* jshint node: true */
+/*jshint esversion: 8 */
+
+"use strict";
+
+const AWS = require('aws-sdk');
+
+const ERR_NAME_INVALID_REQUEST = "InvalidRequestException";
+const DOMAIN_STATUS_PENDING_VERIFICATION = "pending_certificate_dns_validation";
+const ATTEMPTS = 10;
+
+let defaultSleep = function (ms) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+};
+let sleep = defaultSleep;
+let appRoute53Client, appRunnerClient, appHostedZoneID;
+
+/**
+ * Upload a CloudFormation response object to S3.
+ *
+ * @param {object} event the Lambda event payload received by the handler function
+ * @param {object} context the Lambda context received by the handler function
+ * @param {string} responseStatus the response status, either 'SUCCESS' or 'FAILED'
+ * @param {string} physicalResourceId CloudFormation physical resource ID
+ * @param {object} [responseData] arbitrary response data object
+ * @param {string} [reason] reason for failure, if any, to convey to the user
+ * @returns {Promise} Promise that is resolved on success, or rejected on connection error or HTTP error response
+ */
+function report (
+    event,
+    context,
+    responseStatus,
+    physicalResourceId,
+    responseData,
+    reason
+) {
+    return new Promise((resolve, reject) => {
+        const https = require("https");
+        const { URL } = require("url");
+
+        let reasonWithLogInfo = `${reason} (Log: ${context.logGroupName}${context.logStreamName})`;
```

Comment:
it should be `${reason} (Log: ${context.logGroupName}/${context.logStreamName})` :( sorry would you mind helping me modify it in the other lambdas as well?

Reply:
donee